build.sbt

```diff
@@ -25,33 +27,37 @@ versionWithGit ++ versionSettings
 
 val sprayJsonV = "1.3.2"
 
-val lenthallV = "0.21"
+val lenthallV = "0.23-665daf8-SNAP"
 
 resolvers ++= List(
   "Broad Artifactory Releases" at "https://artifactory.broadinstitute.org/artifactory/libs-release/"
 )
 
-libraryDependencies ++= Seq(
-  "org.broadinstitute" %% "lenthall" % lenthallV,
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.4.0",
-  "io.spray" %% "spray-json" % sprayJsonV,
-  /*
-  Exclude test framework cats-laws and its transitive dependency scalacheck.
-  If sbt detects scalacheck, it tries to run it.
-  Explicitly excluding the two problematic artifacts instead of including the three (or four?).
-  https://github.com/typelevel/cats/tree/v0.7.2#getting-started
-   */
-  "org.typelevel" %% "cats" % "0.7.2"
-    exclude("org.typelevel", "cats-laws_2.11")
-    exclude("org.typelevel", "cats-kernel-laws_2.11"),
-  "commons-codec" % "commons-codec" % "1.10",
-  "commons-io" % "commons-io" % "2.5",
-  "org.apache.commons" % "commons-lang3" % "3.4",
-  "com.github.pathikrit" %% "better-files" % "2.16.0",
-  //---------- Test libraries -------------------//
-  "org.scalatest" %% "scalatest" % "3.0.1" % Test,
-  "org.pegdown" % "pegdown" % "1.6.0" % Test
-)
+libraryDependencies ++= {
+  // the if doesn't work if we put this up with the other fooV's
+  val betterFilesV = if (scalaBinaryVersion.value == "2.11") "2.17.1" else "3.0.0"
```

Comment:
We can make it 2.17.1 for both here, I just thought it was neat that you can split versions. It might also be the case at some point that we'll need to take advantage of this trick at some point. But at the moment it's not necessary if you'd rather not see it.